rudder-web/src/main/webapp/secure/administration/policyServerManagement.html

```diff
@@ -234,6 +234,92 @@ <h3>Protocol</h3>
     <div id="agentPolicyMode" class="lift:administration.PropertiesManagement.agentPolicyMode"></div>
     <div id="complianceMode" class="lift:administration.PropertiesManagement.complianceMode"></div>
 
+    <div class="inner-portlet">
+      <div class="page-title">Unexpected reports interpretation</div>
+      <div class="portlet-content">
+        <div class="lift:administration.PropertiesManagement.unexpectedReportInterpretation" id="unexpectedReportInterpretation">
+        <div class="tw-bs">
+        <form class="lift:form.ajax form-horizontal tw-bs" >
+          <div class="col-lg-12 callout-fade callout-warning">
+            <div class="marker">
+              <span class="glyphicon glyphicon-info-sign"></span>
+            </div>
+            The two following settings affect the interpretation given to some type of unexpected reports when computing compliance.
+
+            These options will take effects when the next reports are received from a node or if you "clear caches" below.
+          </div>
+          <div class="col-lg-12 callout-fade callout-warning">
+            <div class="marker">
+              <span class="glyphicon glyphicon-info-sign"></span>
+            </div>
+            <dl>
+              <dt>Duplicated reports</dt>
+              <dd>
+                Due to the underlying protocol used to send compliance reports back to the policy server (syslog),
+                it may happen that some reports, for an unitary control point, are duplicated. In that case, compliance for
+                the corresponding element will be "unexpected": Rudder was awaiting one report, but it got two.
+                The chance to have a real error reported as a duplicated message are very low because you should
```

Comment:
The risk to have a real error reported as a duplicated messages is very low, as messages need to have the same timestamp and information message to be considered duplicated on a given node